.env

```diff
@@ -1 +1,2 @@
-BROWSER=none
\ No newline at end of file
+NEXT_PUBLIC_BASE_PATH=/sosialhjelp/innsyn
```

Comment:
kan denne merges inn i .env.production?
BROWER=none er fra cra, så den kan slettes

Reply:
Setter den her så den gjelder for både local og production